scss/chat.scss

```diff
@@ -23,6 +23,22 @@ body.has-full-page-chat {
 
 .chat-drawer-outlet-container {
   z-index: z("composer", "content");
+
+  .peek-mode-active & {
+    padding-bottom: 0;
+    left: unset;
+    right: var(--main-grid-gap);
+
+    &:has(.is-expanded) {
+      z-index: 701;
```

Comment:
can't we use one of z-index vars?

Reply:
no, they don't fit

Reply:
should we add this case in the core list?

Reply:
Another pattern I've seen is to take a core one, and +1 to it. Not perfect, but I think it's clearer than having a ["magic number"](https://en.wikipedia.org/wiki/Magic_number_(programming)#:~:text=In%20computer%20programming%2C%20a%20magic,replaced%20with%20a%20named%20constant)

Reply:
Yes I think the +1 pattern is fine